PyInstaller/depend/imphook.py

```diff
@@ -301,6 +301,9 @@ def __init__(self, module_graph, module_name, hook_filename,
         # Attributes subsequently defined by the _load_hook_module() method.
         self._hook_module = None
 
+        # Pyinstaller exceptions
+        self.exceptions = ExecCommandFailed(self.module_name, self.hook_filename)
```

Comment:
What's this doing here???

Reply:
This is instantiating the class` ExecCommandFailed()` in exceptions.py

Reply:
This is useless at all, since all the data (exception class, self.module_name, self.hook_filename) is available when raising the exception.